tf-modules/cumulus-test-cleanup/.gitignore

```diff
@@ -0,0 +1,7 @@
+.terraform
```

Comment:
Right now we're publishing all of the tf-modules in the cumulus .zip bundle.    Is this something we should exclude from the .zip file?   If so, should we consider putting it in another location altogether and/or explicitly excluding it from the publish script?

Reply:
Yes we should exclude it. I didn't realize we were publishing all of them. I'm really not intending this to be user-facing or deployed with Cumulus at all. What do you think is the best way to go about this?

Reply:
They're all being included in the deployment .zip file to ensure all subpackages are included.  

*Probably* the sanest thing to do here is to create a subdirectory for maintenance packages that can be included and update L24 of create-release.sh to exclude it.       At some point when we tear apart the monolith TF module structure we could revisit it.

Reply:
I moved these to an `internal` subdirectory with a README saying what it's for and updated line 24. Please double check to see if that seems right.

Reply:
https://github.com/nasa/cumulus/pull/1452/commits/a782bb33ba8c6d6866db4d59e0e709fae132ee4e